.build/parent-pom-template.xml

```diff
@@ -1182,6 +1182,11 @@
         <artifactId>lucene-analysis-common</artifactId>
         <version>9.7.0</version>
       </dependency>
+      <dependency>
+        <groupId>io.github.jbellis</groupId>
+        <artifactId>jvector</artifactId>
+        <version>0.9.2</version>
```

Comment:
1.0.2 has a bunch of bug fixes, IIRC the api changed slightly from 0.9.x but it's an easy change